packages/arakoon/arakoon.1.9.17/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "1.2"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+authors: ["Romain Slootmaekers <romain.slootmaekers@openvstorage.com>"
+          "Jan Doms <jan.doms@openvstorage.com>"]
+homepage: "https://github.com/openvstorage/arakoon"
+bug-reports: "https://github.com/openvstorage/arakoon/issues"
+license: "Apache 2.0"
+dev-repo: "https://github.com/openvstorage/arakoon.git"
+build: [ [make] ]
+
+install: [make "install"   "OCAML_LIBDIR=%{prefix}%/lib" "PREFIX=%{prefix}%"]
+
+remove:  [
+  [make "uninstall_client" "OCAML_LIBDIR=%{prefix}%/lib" "PREFIX=%{prefix}%"]
+  ["rm" "-rf" "%{bin}%/arakoon"]
+]
+
+depends: [
+  "ocamlfind" {build}
+  "bisect" {test & >= "1.3"}
+  "camltc" {>= "0.9.4"}
+  "camlbz2" {>= "0.6.0"}
+  "conf-libev" {>= "4-11"}
+  "lwt" {>= "3.0.0"}
+  "lwt_ssl" {>= "1.0.1"}
+  "ocplib-endian" {>= "1.0"}
+  "ssl" {>= "0.5.3"}
+  "snappy" {>= "0.1.0"}
+  "quickcheck" {>= "1.0.2"}
+  "core" {>="0.9.0"}
+  "ocamlbuild" {build}
+  "redis" {= "0.3.3"}
+  "uri" {>= "1.9.2"}
+]
+available: [ocaml-version >= "4.03.0" & opam-version >= "1.2" & os != "darwin" ]
+
+
+depexts:[
+  [[ "ubuntu"] ["help2man"] ]
```

Comment:
could you also add an equivalent line for `debian` here?

```
[ ["debian"] ["help2man"]]
```

and the CI should pass.